src/components/Combobox/Combobox.jsx

```diff
@@ -93,10 +94,10 @@ const Combobox = forwardRef(
         const optionsByCategories = getOptionsByCategories(filterdOptions, categories, filterValue);
 
         let index = 0;
-        return Object.keys(optionsByCategories).map(categoryId => {
+        return Object.keys(optionsByCategories).map((categoryId, categoryIndex) => {
           return (
             <div role="group" aria-labelledby={`combox-category-${categoryId}`} key={categoryId}>
-              <ComboboxCategory category={categories[categoryId]} />
+              <ComboboxCategory category={categories[categoryId]} divider={categoriesDivider && categoryIndex !== 0} />
```

Comment:
the divider should be a separate unit from the category itself. let's draw it here instead of pass it to the ComboboxCategory

Reply:
V